types/fulu/block.yaml

```diff
@@ -0,0 +1,127 @@
+        allOf:
+          - $ref: "../primitive.yaml#/Signature"
+          - description: "The RANDAO reveal value provided by the validator."
+      eth1_data:
+        $ref: "../phase0/eth1.yaml#/Phase0/Eth1Data"
+      graffiti:
+        $ref: "../primitive.yaml#/Graffiti"
+      proposer_slashings:
+        type: array
+        items:
+          $ref: "../phase0/proposer_slashing.yaml#/Phase0/ProposerSlashing"
+      attester_slashings:
+        type: array
+        items:
+          $ref: "./attester_slashing.yaml#/Electra/AttesterSlashing"
+      attestations:
+        type: array
+        items:
+          $ref: "./attestation.yaml#/Electra/Attestation"
+      deposits:
+        type: array
+        items:
+          $ref: "../phase0/deposit.yaml#/Phase0/Deposit"
+      voluntary_exits:
+        type: array
+        items:
+          $ref: "../phase0/voluntary_exit.yaml#/Phase0/SignedVoluntaryExit"
+      sync_aggregate:
+        $ref: "../altair/sync_aggregate.yaml#/Altair/SyncAggregate"
+      bls_to_execution_changes:
+        type: array
+        items:
+          $ref: "../capella/bls_to_execution_change.yaml#/Capella/SignedBLSToExecutionChange"
+      blob_kzg_commitments:
+        type: array
+        items:
+          $ref: "../primitive.yaml#/KZGCommitment"
+
+  BeaconBlockBody:
+    allOf:
+      - $ref: "#/Fulu/BeaconBlockBodyCommon"
+      - type: object
+        required: [execution_payload, execution_requests]
+        properties:
+          execution_payload:
+            $ref: "../fulu/execution_payload.yaml#/Fulu/ExecutionPayload"
+          execution_requests:
+            $ref: "./electra/execution_requests.yaml#/Electra/ExecutionRequests"
+
+  BeaconBlock:
+    # TODO: update link once confirmed.
+    description: "The [`BeaconBlock`](https://github.com/ethereum/consensus-specs/blob/v1.5.0-beta.2/specs/phase0/beacon-chain.md#beaconblock) object from the CL Fulu spec."
+    allOf:
+      - $ref: "../altair/block.yaml#/Altair/BeaconBlockCommon"
+      - type: object
+        required: [body]
+        properties:
+          body:
+            $ref: "#/Fulu/BeaconBlockBody"
+
+  SignedBeaconBlock:
+    type: object
+    # TODO: update link once confirmed.
+    description: "The [`SignedBeaconBlock`](https://github.com/ethereum/consensus-specs/blob/v1.5.0-beta.2/specs/phase0/beacon-chain.md#signedbeaconblock) object envelope from the CL Fulu spec."
+    required: [message, signature]
+    properties:
+      message:
+        $ref: "#/Fulu/BeaconBlock"
+      signature:
+        $ref: "../primitive.yaml#/Signature"
+
+  BlindedBeaconBlockBody:
+    # TODO: update link once confirmed.
+    description: "A variant of the [`BeaconBlockBody`](https://github.com/ethereum/consensus-specs/blob/v1.5.0-beta.2/specs/electra/beacon-chain.md#beaconblockbody) object from the CL Fulu spec, which contains a transactions root rather than a full transactions list."
+    allOf:
+      - $ref: "#/Electra/BeaconBlockBodyCommon"
```

Comment:
```suggestion
      - $ref: "#/Fulu/BeaconBlockBodyCommon"
```

Reply:
No longer needed as we are not proposing a new `ExecutionPayload` with `ProofVersion` any more. `BeaconBlock` should stay the same as Electra